sdk/storage/storage-file-share/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Release History
 
-## 12.7.0 (Unreleased)
+## 12.7.0-beta.1 (Unreleased)
 
 ### Features Added
+
+- Updated Azure Storage Service API version to 2020-10-02.
 - With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
 - Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features
+- Added support for OAuth in copying source in `ShareFileClient.uploadRangeFromURL()`.
```

Comment:
mention that this only work for Blob source

Reply:
Added